### PR TITLE
Added Polish translations

### PR DESCRIPTION
**Pull request by bpluta**

### CODE REVIEW

File: selfdrive/ui/translations/main_pl.ts [hunk 1]
```diff
@@ -0,0 +1,1336 @@
...
+        <source>Cancel</source>
+        <translation>Anuluj</translation>
+    </message>
+    <message numerus="yes">
+        <location filename="../qt/widgets/input.cc" line="168"/>
+        <source>Need at least %n character(s)!</source>
+        <translation type="unfinished">
+            <numerusform></numerusform>
+            <numerusform></numerusform>
+            <numerusform></numerusform>
```
Comment (sshane) [inline selfdrive/ui/translations/main_pl.ts:315]: Do you mind adding the new plural forms?

Comment (bpluta) [reply selfdrive/ui/translations/main_pl.ts]: Sure! I have just updated the PR 👍

Comment (sshane) [reply selfdrive/ui/translations/main_pl.ts]: Thanks!

**Comment by sshane** [comment]
Opened relevant issue: https://github.com/commaai/openpilot/issues/25492

**Comment by sshane** [comment]
Thanks for the translations! Merged without adding to the UI just yet, as some texts are longer than our design allows for. You're welcome to add the line back locally to daily drive with it, and possibly consider shorter texts if they're good changes

**Comment by 0xF16** [comment]
Was about to do the Polish translation as well. Had 50 strings left to translate locally :D Definitely, I'll be testing it and proposing some changes if I see them fit.

**Pull request merged**